src/canvas.class.js

```diff
@@ -1216,18 +1220,18 @@
     /**
      * @private
      */
-    _searchPossibleTargets: function(objects, pointer) {
```

Comment:
even if those 2 methods are private, we could still document them
```
Function used to search inside objects an object that contains pointer in bounding box or that contains pointerOnCanvas when painted
@param [fabric.Object] objects array to look into
...
```

can you complete the JSDOC information for _searchPossibleTargets  and checkTarget ?

Reply:
sure